chpl/chpl-api/e2e/data/chplIds-test-data.json

```diff
@@ -0,0 +1,17 @@
+[
+	{
+		"certifiedProductId": 3321,
+		"certifiedlistings2015": "15.04.04.2214.Proc.14.00.0.180427",
+		"certifiedlistings2014": "CHP-028396"
```

Comment:
I'm very confused by this file. This looks like it should be one listing, but there's no way for a listing to have two different CHPL Product Numbers in two different formats. The Listing with the database ID 3321 _must_ be either a 2014 or a 2015 Listing. If it's a 2014, there's a chance it has the CHP-nnnnn style Product Number, but if it's a 2015 it's impossible to have that style

Reply:
Done